hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieFileGroup.java

```diff
@@ -106,13 +116,27 @@ public String getPartitionPath() {
    */
   private boolean isFileSliceCommitted(FileSlice slice) {
     String maxCommitTime = lastInstant.get().getTimestamp();
-    return timeline.containsOrBeforeTimelineStarts(slice.getBaseCommitTime())
-        && HoodieTimeline.compareTimestamps(slice.getBaseCommitTime(),
+    return timeline.containsOrBeforeTimelineStarts(slice.getBaseInstantTime())
+        && HoodieTimeline.compareTimestamps(slice.getBaseInstantTime(),
         maxCommitTime,
         HoodieTimeline.LESSER_OR_EQUAL);
 
   }
 
+  /**
+   * Get all the the file slices as seen in underlying file-system
+   */
+  public Stream<FileSlice> getAllUnfilteredFileSlices() {
```

Comment:
Can we keep this as getFileSlices(), we can rename the ones which return something filtered as GetFiltered...

Reply:
@n3nash : This method is dangerous as it will return all files due to inflight ingestion/compaction. A cleanup or admin-CLI level commands only should be using it. That was the reason why I did not name this method using the generic name "getFileSlices". Let me know if you think otherwise.

Reply:
I see, getAllFileSlicesIncludingInflight() to make it more clear ? I'm probably reading into this more than needed. I'm fine either ways.

Reply:
Done.

Reply:
@n3nash : Thanks. Updated the PR